python/cmake/Modules/init_python.cmake

```diff
@@ -62,9 +62,11 @@ macro(init_python target_version)
       return()
    endif()
 
-   if (PYTHON_VERSION_MAJOR EQUAL 3)
-      message(WARNING "libecl is not PYTHON 3 compatible. Python wrappers are enabled, but will fail horribly.")
-   elseif (NOT "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}" STREQUAL "${target_version}")
+  if (PYTHON_VERSION_MAJOR EQUAL 3)
+      if (NOT PYTHON_VERSION_MINOR EQUAL 6)
+          message(WARNING "libecl for Python 3 is only tested and verified against Python 3.6. Python wrappers are enabled, but not guaranteed to work.")
+      endif()
+  elseif (NOT "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}" STREQUAL "${target_version}")
       message(WARNING "Need Python version ${target_version}, found version: ${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR} - Python wrappers not enabled")
```

Comment:
I suggest we change this to be explicit on that if MAJOR is 2, we only support `2.7`. Now, that message sounds like we only support `2.7` period.